lib/practice_exercises.rb

```diff
@@ -1,13 +1,39 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def is_palindrome(string)
```

Comment:
👍 

Although since you're creating a new array with `string.downcase.split`, this has O(n) space complexity.